airbyte-integrations/connectors/source-dremio/source_dremio/spec.yaml

```diff
@@ -0,0 +1,19 @@
+documentation_url: "https://docs.airbyte.com/integrations/sources/dremio"
```

Comment:
if this is used for integration testing it should probably go in the integration test folder as `expected_spec.yaml` or something

Reply:
moved the file, thanks for suggestion 👍